packages/webauthn-pcd/package.json

```diff
@@ -0,0 +1,36 @@
+{
+  "name": "@pcd/webauthn-pcd",
+  "version": "0.0.10",
+  "license": "GPL-3.0-or-later",
+  "main": "./index.ts",
+  "types": "./index.ts",
+  "scripts": {
+    "lint": "eslint \"**/*.ts{,x}\"",
+    "test": "ts-mocha --exit test/**/*.spec.ts",
+    "build": "tsc --noEmit"
+  },
```

Comment:
the way we publish packages from this repo is not optimal at the moment: they are compiled on `postinstall`. check out the `scripts` field in the `package.json` of something like the `semaphore-group-pcd` package to see how we do it.